cypress/README.md

```diff
@@ -20,4 +20,73 @@ npm run test:it
 
 ### Run integration tests with real data
 
-Comming soon
+**Run UI**
+
+Before you start. Please [set](../README.md) the environment which is required.
+
+The UI runs against the url which is stored in [the .env file](https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox-ui/blob/main/.env).
+The default value of the `BASE_URL` is set to the DEV environment. It is shared environment. Please use the hash for each name of bridges to protect name conflict.
+
+Then run:
+
+```
+npm run start
+```
+
+**Set credentials**
+
+Get a valid offline [token](https://console.redhat.com/openshift/token).
+
+```
+export CYPRESS_USER=<replce with your value>
+export CYPRESS_PASSWORD=<replce with your value>
+export OPENSHIFT_OFFLINE_TOKEN=<REPLACE WITH YOUR TOKEN>
+export CYPRESS_OB_TOKEN="$(curl -s --insecure -X POST https://sso.redhat.com/auth/realms/redhat-external/protocol/openid-connect/token --header 'Content-Type: application/x-www-form-urlencoded' --data-urlencode 'client_id=cloud-services' --data-urlencode 'grant_type=refresh_token' --data-urlencode "refresh_token=$OPENSHIFT_OFFLINE_TOKEN" | jq --raw-output '.access_token')"
+```
+
+If the test suite fail as Unauthorized than perform again:
+
+```
+export CYPRESS_OB_TOKEN="$(curl -s --insecure -X POST https://sso.redhat.com/auth/realms/redhat-external/protocol/openid-connect/token --header 'Content-Type: application/x-www-form-urlencoded' --data-urlencode 'client_id=cloud-services' --data-urlencode 'grant_type=refresh_token' --data-urlencode "refresh_token=$OPENSHIFT_OFFLINE_TOKEN" | jq --raw-output '.access_token')"
+
+```
+
+**Run E2E tests**
+
+```
+npm run cypress:open:e2e
+```
+
+Clear entities which were added by the test suite.
+
+## Run UI with real data locally
```

Comment:
Do we want to document this within the repo?

Asking because the whole process is hackish.

We already have mocked APIs + `dev` APIs to test new features. Do we need another way of doing this?

Reply:
well it is good question, I can imagine this can be beneficial for testing latest features of backend locally, if you mind we have this documented publicly, maybe we can move it ot some google document, or redhat source page?

Reply:
@kelvah I have doubted to paste or delete this chapter here. As you mention it is `It is not the supported usecase.` I would like to mention it somewhere, because it could be useful. I have no problem to delete it.

Reply:
Thanks @sjamboro 

@jomarko +1 on moving this instructions into a google document.